controls/tabview/styling/content-styling.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Content Styling
+page_title: .NET MAUI TabView Documentation - Content Styling
+description: Learn how to style the Telerik UI for .NET MAUI TabView Content by setting border thickness, color and corner radius.
+slug: tabview-content-styling
+tags: tabview, content, style, maui, dotnet
+position: 4
+---
+
+# .NET MAUI TabView Content Styling
+
+TabView's `ContentStyle` property allows you to style the content area. `TabViewContent` provides the following properties:
```

Comment:
See the comment in `controls/tabview/styling/header-styling.md` for an example of how to add an introductory sentence for the article and then for the list/table.